pom.xml

```diff
@@ -64,17 +64,22 @@
     <properties>
         <revision>2.14</revision>
         <changelist>-SNAPSHOT</changelist>
-        <jenkins.version>2.121.1</jenkins.version>
+        <jenkins.version>2.121.3</jenkins.version>
         <java.level>8</java.level>
         <workflow-cps-plugin.version>2.69</workflow-cps-plugin.version>
```

Comment:
Where in here are we picking up the new `script-security`? Or were we already picking it up?

Reply:
We're using 1.58 after you updated it in https://github.com/jenkinsci/workflow-cps-global-lib-plugin/pull/67. I will go ahead and bump to 1.60 though.

Reply:
Done in f9010950dd71060542d8b4dd61267a17ab6342df.